framework/src/controller/bus.ts

```diff
@@ -32,6 +32,7 @@ interface BusConfiguration {
 		readonly enable: boolean;
 		readonly mode: string;
 		readonly port: number;
+		readonly host?: string;
```

Comment:
why is it optional? we have a default host rite?

Reply:
For `WSServer` class `host` is optional, I think should be the same behavior for `bus` but not sure ?
Although, I think logically, it's fine either way (optional or not).